README.md

```diff
@@ -4,12 +4,14 @@ This is a capistrano v3 plugin that integrates Unicorn tasks into capistrano dep
 
 ### Gotchas
 
-- The `unicorn:start` task invokes unicorn as `bundle exec unicorn`.
-
 - When running tasks not during a full deployment, you may need to run the `rvm:hook`:
 
     `cap production rvm:hook unicorn:start`
 
+- To use unicorn with bundler, you'll have to add the `capistrano-bundler` gem, and add the unicorn command to the bundle bins:
+    
+    `set :bundle_bins, fetch(:bundle_bins, []).push('unicorn')`
```

Comment:
but it work's: `set :rvm_map_bins, fetch(:rvm_map_bins).to_a.concat(%w(unicorn))` or `set :rbenv_map_bins, fetch(:rbenv_map_bins).to_a.concat(%w(unicorn))` or for `: chruby_map_bins` or for `bundle_bins`